great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -105,12 +105,24 @@ def _parse_batching_regex_string(
     ) -> re.Pattern:
         return Datasource.parse_batching_regex_string(batching_regex=batching_regex)
 
-    def batch_request_options_template(
+    @property
```

Comment:
Why are we changing this to a property?

Reply:
This was discussed more in-depth internally, but the short answer is because it will never accept arguments, and is just a reference for the user.